### ISSUE TITLE
Create ORDO subset ROBOT template: Update goal to depend on component

### ISSUE THREAD

**Issue by twhetzel** [monarch-initiative/mondo-ingest#511]
Based on the work to update the subsets in Mondo based on information from ORDO, this question about whether to update the URL for the `component-download` came up. See https://github.com/monarch-initiative/mondo-ingest/pull/510/files#r1588599701

Response from Nico:
---
Look at
```
$(COMPONENTSDIR)/ordo.owl: $(TMPDIR)/ordo_relevant_signature.txt config/properties.txt | component-download-ordo.owl
	if [ $(COMP) = true ]; then $(ROBOT) remove -i $(TMPDIR)/component-download-ordo.owl.owl --select imports \
		merge ...
```
It is better to:

depend in component-download-ordo.owl
And use $(TMPDIR)/component-download-ordo.owl.owl
to ensure we are using the same version as the rest of the pipeline

**Comment by joeflack4** [comment]
Yeah, I just saw that comment. I'm a bit confused by Nico's response.

My thought is that, as I suggested, all that needs to be updated is `mondo-ingest-odk.yaml`, and just update the URL in this section...
```yml
components:
  products:
    ...
    - filename: ordo.owl
      source: URL
```
...replacing `URL` with the one we used for #510: `https://www.orphadata.com/data/ontologies/ordo/last_version/ORDO_en_4.4.owl`

If we do this, then we would run `sh run.sh make update_repo`, and this will update the `Makefile` such that the `component-download-ordo.owl` will now use this updated URL.

**Comment by matentzn** [comment]
> My thought is that, as I suggested, all that needs to be updated is mondo-ingest-odk.yaml, and just update the URL in this section...

The config already has the correct URL:

https://github.com/monarch-initiative/mondo-ingest/blob/e093456bdb193f609b71e3ba5ce7baab3834fcd5/src/ontology/mondo-ingest-odk.yaml#L43

It points to the _latest version of raw ORDO_, not to the versioned PURL you are currently using.

Is there any other objection to my suggested plan (just depending in the existing mirror instead of creating a new one)?

**Comment by joeflack4** [comment]
@matentzn Ah ok, I think the source of my confusion here is this:

_[Trish wrote](https://github.com/monarch-initiative/mondo-ingest/issues/509#issuecomment-2091263043):_
> (1) is this the location you want for the ORDO file (this is the value from OLS for the Ontology IRI): https://www.orphadata.com/data/ontologies/ordo/last_version/ORDO_en_4.4.owl

_[Nico wrote](https://github.com/monarch-initiative/mondo-ingest/issues/509#issuecomment-2091383147)_:
> (1) yes, exactly!

So I figured there was something wrong with http://www.orphadata.org/data/ORDO/ordo_orphanet.owl.

But I think that's not the case. I think @matentzn you just misread the URL that Trish was linking there? If that's the case, then yes, I agree, for #510 I should change it to depend on `component-download-ordo.owl` instead.

Do I understand correctly now?

**Comment by matentzn** [comment]
Yes, sounds right! Sorry for causing the confusion!

**Comment by joeflack4** [comment]
Ok, @twhetzel this is super easy-peasy then. I need to:

- [ ] @joeflack4 update #510 to depend on `component-download-ordo.owl` instead of https://www.orphadata.com/data/ontologies/ordo/last_version/ORDO_en_4.4.owl

**Comment by matentzn** [comment]
Both very important questions, lets pick these up when we resume the work on this

**Comment by joeflack4** [comment]
Update:
- Status of this issue: I've addressed it's concern in a commit to #510 
- Question 1 above: addressed
- Question 2 above: split into a new issue #514, so we don't need to worry about it here, and it [Nico also said](https://github.com/monarch-initiative/mondo-ingest/pull/510#discussion_r1595178865) it's out of scope for #510

So this issue can be closed when #510 is merged.

**Comment by joeflack4** [comment]
Addressed by:
- #510
